packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTableSet.tsx

```diff
@@ -193,14 +193,14 @@ export class ApiReferencesTableSet extends React.Component<IApiReferencesTableSe
     const enumMembers: IApiEnumProperty[] = [];
 
     const members: IEnumTableRowJson[] = table.members as IEnumTableRowJson[];
-    for (let k = 0; k < members.length; k++) {
+    members.map((member: IEnumTableRowJson) => {
```

Comment:
`map` is meant for making a 1-1 mapping from an existing array to a new array, so it should return something rather than doing an operation with a side effect (`forEach` would be more appropriate in that case). So you can simplify the code like this:
```ts
const enumMembers: IApiEnumProperty[] = members.map((member: IEnumTableRowJson) => ({
  description: member.description,
  name: member.name,
  value: member.value
});
```
(The extra parentheses around the returned object are needed to keep it from being interpreted as a function body.)